Assets/Scripts/Gameplay/GameState/ClientCharSelectState.cs

```diff
@@ -103,9 +103,14 @@ enum LobbyMode
         protected override void Awake()
         {
             base.Awake();
-
             Instance = this;
-            CharSelectData = GetComponent<CharSelectData>();
+
+            // TODO inject or find another way to find CharSelectData
+            // TODO CharSelectData should directly be in ServerCharSelectState and both client and server should be in same gameplay assembly
```

Comment:
> and both client and server should be in same gameplay assembly

Isn't it the case now?

Reply:
amended the wording of the comment, as for moving charselectdata - let it be a small self-contained PR in followup 🙏